internal/clientcache/internal/cache/options.go

```diff
@@ -42,6 +45,14 @@ func WithUpdateLastAccessedTime(b bool) Option {
 	}
 }
 
+// WithDuration provides an option for specifying a duration
```

Comment:
A duration of what? Without any context I don't know what this function does. Could you expand this doc string?